src/Jellyfin.Plugin.Dlna/Jellyfin.Plugin.Dlna.csproj

```
@@ -18,6 +18,8 @@
     <ItemGroup>
       <EmbeddedResource Include="Configuration\config.html" />
       <EmbeddedResource Include="Configuration\config.js" />
+      <EmbeddedResource Include="Images\*.jpg" />      
```

Trailing white space
```suggestion
      <EmbeddedResource Include="Images\*.jpg" />
```